### PR TITLE
Public demoshop

### PR DESCRIPTION
**Pull request by boris-brtan**
| Q             | A
| ------------- | ---
|Description, reason for the PR| we want to add new domain with german locale
|New feature| Yes 
|Fixes issues| 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: src/Shopsys/ShopBundle/DataFixtures/Demo/MultidomainArticleDataFixture.php [hunk 1]
```diff
@@ -54,9 +54,7 @@ public function __construct(
      */
     public function load(ObjectManager $manager)
     {
-        foreach ($this->domain->getAllIdsExcludingFirstDomain() as $domainId) {
-            $this->loadForDomain($domainId);
-        }
+        $this->loadForDomain(2);
```
Comment (TomasLudvik) [inline src/Shopsys/ShopBundle/DataFixtures/Demo/MultidomainArticleDataFixture.php]: I do not see any reason for this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,6 +21,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - prices of products are recalculated for logged customers based on custom discount
     - discount works also for companies with multiple users
     - `filterFormMacro.html.twig` now counts with discount coeficient
+- [#55 - Third domain with german locale](https://github.com/shopsys/demoshop/pull/55)
+    - domains and domains_url search index routing were added for third domain
+    - datafixtures were fixed to generate only supported multidomain demodata
+    - CategoryRepositoryTest was fixed for third domain
+    - translations were copied from en language and prefixed with (de)
```
Comment (TomasLudvik) [inline CHANGELOG.md:27]: US says that prefix should be "DE_" why did you chose something else?

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (boris-brtan) [reply CHANGELOG.md:27]: @LukasHeinz changed his opinion today

File: src/Shopsys/ShopBundle/DataFixtures/Demo/MultidomainArticleDataFixture.php (hunk 1 above)
Comment (boris-brtan) [reply src/Shopsys/ShopBundle/DataFixtures/Demo/MultidomainArticleDataFixture.php]: else it failed on non existent csv files

Comment (TomasLudvik) [reply src/Shopsys/ShopBundle/DataFixtures/Demo/MultidomainArticleDataFixture.php]: So you should create those files and prefix everything with (de) right?

**Review by TomasLudvik — APPROVED** [review]

**Pull request merged**